lib/studies/reply/app.dart

```diff
@@ -38,6 +44,42 @@ class ReplyApp extends StatelessWidget {
         routes: <String, WidgetBuilder>{
           homeRoute: (context) => const AdaptiveNav(),
         },
+        onGenerateRoute: (settings) {
+          switch (settings.name) {
+            case homeRoute:
+              {
+                return MaterialPageRoute<void>(
+                  builder: (context) {
+                    return const AdaptiveNav();
+                  },
+                );
+              }
+              break;
```

Comment:
nit: breaks, default case and returning `null` are unnecessary

Reply:
I get a warning when removing the `default` case `This function has a return type of 'Route<dynamic>, but doesn't end with a return statement`. I was thinking the gallery tests may flag this warning.

Reply:
Oh right, yeah the analyzer would flag it. I wonder if you return null does it still complain? One or the other should make the analyzer happy

Reply:
Yeah I ended up just returning null at the end. Thanks!